webpack/assets/javascripts/react_app/components/common/Fill/GlobalFill.js

```diff
@@ -0,0 +1,8 @@
+import { registerFillComponent } from '../Fill/FillActions';
+import store from '../../../redux';
+
+export const addGlobalFill = (slotId, fillId, component, weight) => {
```

Comment:
Don't you want to allow `globalFills` with `overrideProps`?

Reply:
see at the documentation, the component can be a also a props object